jvb/src/test/kotlin/org/jitsi/videobridge/ConferenceTest.kt

```diff
@@ -37,50 +33,28 @@ class ConferenceTest : ConfigTest() {
 
     init {
         val name = JidCreate.entityBareFrom("roomName@somedomain.com")
-        withNewConfig(newConfigOctoEnabled(), loadDefaults = true) {
-            octoRelayServiceProvider().get()?.start()
-        }
 
         context("Adding local endpoints should work") {
-            with(Conference(videobridge, "id", name, Conference.GID_NOT_SET, null, false, false)) {
+            with(Conference(videobridge, "id", name, null, false, false)) {
                 endpointCount shouldBe 0
                 createLocalEndpoint("abcdabcd", true, false) // TODO cover the case when it's true
                 endpointCount shouldBe 1
                 debugState.shouldBeValidJson()
             }
         }
         context("Enabling octo should fail when the GID is not set") {
```

Comment:
Is this test meaningful now?

Reply:
I removed/updated this test in a later commit (sorry)